<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPlaylist</class>
 <widget class="QWidget" name="SettingsPlaylist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-desktop">
    <normaloff/>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-79</y>
        <width>408</width>
        <height>415</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupDefaultPlaylist">
         <property name="title">
          <string>Default playlist</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="BrowseWidget" name="playlist" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupPlaylistRefresh">
         <property name="title">
          <string>Network refresh</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="checkRefresh">
            <property name="text">
             <string>Refresh playlist on startup</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelRefresh">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Playlist URL:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editUrl">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelRefreshInfo">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&lt;i&gt;Updating will not change your playlist order.&lt;/i&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupRadio">
         <property name="title">
          <string>Channel types categories</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="labelRadio">
            <property name="text">
             <string>Default radio category:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="editRadio"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelHd">
            <property name="text">
             <string>Default HD category:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="editHd"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupUdpxy">
         <property name="title">
          <string>UDPXY support</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkUdpxy">
            <property name="text">
             <string>Enable UDPXY support</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="editUdpxyUrl">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="editUdpxyPort">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>1234</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelUdpxyUrl">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Router IP or name:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelUdpxyPort">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Router port:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrowseWidget</class>
   <extends>QWidget</extends>
   <header>common/BrowseWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>checkRefresh</tabstop>
  <tabstop>editUrl</tabstop>
  <tabstop>editHd</tabstop>
  <tabstop>editRadio</tabstop>
  <tabstop>checkUdpxy</tabstop>
  <tabstop>editUdpxyUrl</tabstop>
  <tabstop>editUdpxyPort</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkRefresh</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelRefresh</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkRefresh</sender>
   <signal>toggled(bool)</signal>
   <receiver>editUrl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkRefresh</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelRefreshInfo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkUdpxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>editUdpxyUrl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkUdpxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelUdpxyPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkUdpxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>editUdpxyPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkUdpxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelUdpxyUrl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
